src/libostree/ostree-sysroot-deploy.c

```diff
@@ -808,6 +809,7 @@ static gboolean
 write_origin_file_internal (OstreeSysroot         *sysroot,
                             OstreeSePolicy        *sepolicy,
                             OstreeDeployment      *deployment,
+                            gboolean               if_not_exists,
```

Comment:
I'm confused why we need this. Why not just unconditionally overwrite it in `finalize`? Then even in the stage path, we're sure of using the right label for it post-merge.